site/docs/voice/bxml/conference.mdx

```diff
@@ -45,7 +45,7 @@ BXML containing unsupported verbs will be rejected completely.
 
 | Name        | Description |
 |:------------|:------------|
-| name        | The name of the conference with alphanumeric characters and the symbols `-`, `_`, and `.` with maximum length of 100 characters. |
+| name        | The name of the conference. Can contain up to `100` characters of letters, numbers, and the symbols `-`, `_`, and `.`/ |
```

Comment:
Why is this trailing `/` here?

Reply:
Typo

Reply:
I think the real question is should I put a second period after that one